compiler/ir/backend.js/src/org/jetbrains/kotlin/ir/backend/js/lower/EnumClassLowering.kt

```diff
@@ -125,9 +125,6 @@ class EnumClassConstructorTransformer(val context: CommonBackendContext, private
     private val loweredEnumConstructors = HashMap<IrConstructorSymbol, IrConstructor>()
 
     fun transform(): List<IrDeclaration> {
-        // Make sure InstanceInitializer exists
```

Comment:
@skuzmich Please take a look for js/lower/EnumClassLowering.kt  changes

Reply:
@sfs thanks a lot for dealing with this workaround. LGTM.